Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp

```diff
@@ -292,6 +292,9 @@ bool WebLoaderStrategy::tryLoadingUsingURLSchemeHandler(ResourceLoader& resource
     if (!webPage || !webFrame)
         return false;
 
+    if (resourceLoader.request().url().protocolIsAbout() && is<SubresourceLoader>(resourceLoader))
```

Comment:
Is this still needed?

Reply:
It seems we should keep this as is. I tried several solution that handling about:blank, returning empty 200 response to all request or returning error, some tests are still failing.